### PR TITLE
Update playground

### PR DESCRIPTION
**Pull request by RuiAAPeres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,7 +347,17 @@ Once you’ve set up your project, check out the [Framework Overview][] for
 a tour of ReactiveCocoa’s concepts, and the [Basic Operators][] for some
 introductory examples of using it.
 
+## Playground
```
Comment (NachoSoto) [inline README.md]: 😍

File: ReactiveCocoa.playground/Pages/SignalProducer.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -699,3 +699,34 @@ scopedExample("`flatMapError`") {
...
+ Logs all events that the receiver sends.
+ By default, it will print to the standard output.
+ */
+scopedExample("`log events`") {
+    
+    let baseProducer = SignalProducer<Int, NoError>(values: [1,2,3,4,42])
+    
+    baseProducer
+        .logEvents(identifier: "Playground is fun!")
+        .startWithNext { _ in
```
Comment (NachoSoto) [inline ReactiveCocoa.playground/Pages/SignalProducer.xcplaygroundpage/Contents.swift]: Maybe just call `.start()`?

File: ReactiveCocoa.playground/Pages/SignalProducer.xcplaygroundpage/Contents.swift [hunk 3]
```diff
@@ -699,3 +699,34 @@ scopedExample("`flatMapError`") {
...
+ ### `logEvents`
+ Logs all events that the receiver sends.
+ By default, it will print to the standard output.
+ */
+scopedExample("`sampleWith`") {
+    
+    let producer = SignalProducer<Int, NoError>(values: [1,2,3,4])
+	let sampler = SignalProducer<String, NoError>(values: ["a", "b"])
+				
+	let result = producer.sampleWith(sampler)
```
Comment (NachoSoto) [inline ReactiveCocoa.playground/Pages/SignalProducer.xcplaygroundpage/Contents.swift]: Indentation here seems wrong

**Comment by mdiep** [comment]
👍

**Pull request merged**